doghouse/server/github.go

```diff
@@ -0,0 +1,75 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/bradleyfalzon/ghinstallation"
+	"github.com/google/go-github/github"
+	"github.com/haya14busa/reviewdog/doghouse/server/storage"
+)
+
+type NewGitHubClientOption struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type NewGitHubClientOption should have comment or be unexported